src/main/java/edu/cornell/kfs/concur/rest/resource/ConcurAccountValidationResource.java

```diff
@@ -28,6 +29,7 @@ public Response validate(ConcurEventNotificationDTO concurEventNotificationDTO)
         try {
             concurEventNotification = getConcurEventNotificationConversionService().convertConcurEventNotification(concurEventNotificationDTO);
         } catch (ParseException e) {
+            LOG.error(e.getMessage());
```

Comment:
This should probably include the function name in the error message.  Also, should the error be logged as well as the message?